### ISSUE TITLE
Cannot run the demo

### ISSUE THREAD

**Issue by ghost** [llSourcell/Landing-a-SpaceX-Falcon-Heavy-Rocket#1]
When I try to run ppo.py, it fails to load GymEnvironment. I'm not sure what to do. Any suggestions?

 File "ppo.py", line 8, in <module>
    from agents import GymEnvironment

**Comment by buddhashrestha** [comment]
Try to import it via python shell first.
Example: 
`>>>` from agents import GymEnvironment
if it shows error in that, then you need to add a system path via python shell to 
you can use:
`>>>` import sys
`>>>` sys.path.insert(0, 'your_path')
try again and it should work

**Comment by dfolz** [comment]
I'm also having the same issue, the sys.path.insert did not work. I opened a python shell in terminal and typed in the path to the Rocket master folder. Any other suggestions?

PS I'm having trouble loading ppo.renderthread

**Comment by buddhashrestha** [comment]
also you need to create an empty file with the name `__init__.py` inside  /Landing-a-SpaceX-Falcon-Heavy-Rocket/ppo folder

**Comment by jojojoy2k** [comment]
New to Python. 
I am getting this error: 
`ModuleNotFoundError: No module named 'ppo.renderthread'; 'ppo' is not a package`

**Comment by jojojoy2k** [comment]
Adding a missing `__init__.py` in the `ppo` folder fixed the issue for me.

**Comment by ghost** [comment]
Yeah, python can't find the ppo files otherwise. Python is loading functions from the python scripts in the ppo folder.

**Comment by ericchai** [comment]
yes,Adding a missing __init__.py in the ppo folder.
